api/locale/fa.yml

```diff
@@ -0,0 +1,21 @@
+fa:
+  direction: rtl
+  views: "بازدید %{number}"
+  seconds-ago:
+    one: "یک ثانیه پیش"
+    many: "ثانیه پیش %{count}"
+  minutes-ago:
+    one: "یک دقیقه پیش"
+    many: "دقیقه پیش %{count}"
+  hours-ago:
+    one: "یک ساعت پیش"
+    many: "ساعت پیش %{count}"
+  days-ago:
+    one: "یک روز پیش"
+    many: "روز پیش %{count}"
+  months-ago:
+    one: "یک ماه پیش"
+    many: "ماه پیش %{count}"
+  years-ago:
+    one: "یک سال پیش"
+    many: "سال پیش %{count}"
```

Comment:
Using rtl as the base direction, currently, it seems the cards display like this:

![image](https://user-images.githubusercontent.com/20955511/195894287-a85d5422-5f15-4ed2-8ad5-9b039f65da65.png)

Based on my understanding and checking examples on YouTube, it seems the numbers should go on the right.

![image](https://user-images.githubusercontent.com/20955511/195894388-cbc583b3-3a51-4de5-99b7-5434c0e4932f.png)

It may seem backwards, but I believe the yml should have the english part on the left to make it display on the right.

Let me know if you agree and feel free to commit this suggestion.

```suggestion
  views: "%{number} بازدید"
  seconds-ago:
    one: "یک ثانیه پیش"
    many: "%{count} ثانیه پیش"
  minutes-ago:
    one: "یک دقیقه پیش"
    many: "%{count} دقیقه پیش"
  hours-ago:
    one: "یک ساعت پیش"
    many: "%{count} ساعت پیش"
  days-ago:
    one: "یک روز پیش"
    many: "%{count} روز پیش"
  months-ago:
    one: "یک ماه پیش"
    many: "%{count} ماه پیش"
  years-ago:
    one: "یک سال پیش"
    many: "%{count} سال پیش"
```